### ISSUE TITLE
Improve error displaying

### ISSUE THREAD

**Issue by chedieck** [PayButton/paybutton#327]
Right now is not even red, also maybe we should blur the QR code or something?

![image](https://github.com/PayButton/paybutton/assets/21281174/8eca90cf-8ce7-4f28-84bf-83320e65dda0)

**Comment by Klakurka** [comment]
Yeah the QR code should be blurred out -- in fact we already do this in at least one case... I'm just blanking on when that is but we can reuse that for whenever there's an error.

**Comment by johnkuney** [comment]
Is there more to do here? Looks like there is a red error message and the QR is blurred now on an error
